BUILDING.md

```diff
@@ -45,7 +45,7 @@ The following build targets help you configure the development environment. If y
 * `test_slow` | Run all non-fast tests.
 * `wheel` | Build wheels in release mode.
 
-The script uses Python 3.7, but you can include `--python=3.6` (or `3.7`, `3.8`, `3.9`) as a argument to change the Python version.
+The script uses Python 3.7, but you can include `--python=3.6` (or `3.7`, `3.8`, `3.9`, `3.10`) as an argument to change the Python version.
```

Comment:
```suggestion
coremltools's code uses Python 3.7, but you can include `--python=3.6` (or `3.7`, `3.8`, `3.9`, `3.10`) as an argument to change the Python version.
```

Reply:
Hmm I don’t think “coremltool’s code uses Python 3.7” is accurate? The code supports multiple Python versions; it’s just that the script(s) use Python 3.7 by default.

Reply:
Oopsie; I wasn't aware of that. Sorry